apps/api/src/app/events/usecases/map-trigger-recipients/map-trigger-recipients.e2e.ts

```diff
@@ -171,10 +178,12 @@ describe('MapTriggerRecipientsUseCase', () => {
 
   describe('When feature enabled', () => {
     beforeEach(() => {
+      process.env.LAUNCH_DARKLY_SDK_KEY = '';
       process.env.FF_IS_TOPIC_NOTIFICATION_ENABLED = 'true';
     });
 
     afterEach(() => {
+      process.env.LAUNCH_DARKLY_SDK_KEY = originalLaunchDarklySdkKey;
```

Comment:
We restore the Launch Darkly SDK key when done with the tests to not influence other test suites.